python/Ganga/GPIDev/Lib/File/GoogleFile.py

```diff
@@ -48,11 +56,11 @@ class GoogleFile(IGangaFile):
                       'localDir': SimpleItem(defvalue="", copyable=1,
                                              doc='local dir where the file is stored, used from get and put methods'),
                       'subfiles': ComponentItem(category='gangafiles', defvalue=[], hidden=1,
-                                                typelist=['Ganga.GPIDev.Lib.File.LCGSEFile'], sequence=1, copyable=0,
+                                                typelist=[GoogleFileStub], sequence=1, copyable=0,
```

Comment:
This is my point. On any `ComponentItem`s we don't need `typelist` as it shouldn't be used anywhere. That attribute can be completely removed from all of them. There aren't many and can be found with some grepping.